packages/module/src/FileDropZone/FileDropZone.tsx

```diff
@@ -27,7 +26,18 @@ const FileDropZone: React.FunctionComponent<FileDropZoneProps> = ({
   const renderDropZone = () => (
     <>
       <MultipleFileUploadMain
-        titleIcon={<UploadIcon />}
+        titleIcon={
+          <svg width="50" height="50" viewBox="0 0 50 50" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+            <path
+              d="M48.1116 47.0088H1.88933C1.597 47.0088 1.31665 47.1249 1.10994 47.3316C0.903236 47.5383 0.787109 47.8187 0.787109 48.111C0.787109 48.4033 0.903236 48.6837 1.10994 48.8904C1.31665 49.0971 1.597 49.2132 1.88933 49.2132H48.1116C48.4039 49.2132 48.6842 49.0971 48.8909 48.8904C49.0977 48.6837 49.2138 48.4033 49.2138 48.111C49.2138 47.8187 49.0977 47.5383 48.8909 47.3316C48.6842 47.1249 48.4039 47.0088 48.1116 47.0088Z"
```

Comment:
the preview surge link on the PR is not working (I think it's a surge outage issue) so I cannot verify unless running locally - do these SVGs work in dark theme? if not, you might need to use a color token as the stroke or fill color. You can see that done in this demo: https://github.com/patternfly/virtual-assistant/blob/v6/packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/IncludingAThemeSensitiveSVG.tsx#L15

Reply:
I set the fill to "currentColor" so it takes whatever fill it currently there. It should work just fine in dark mode, but let me know if you'd rather I used the color token as the stroke or fill color.